pkgs/by-name/kr/kryptor/update.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p curl jq common-updater-scripts
```

Comment:
```suggestion
#!nix-shell --pure -i bash -p bash cacert curl jq common-updater-scripts
```
Nit

Reply:
Why is `bash` needed? (also adding `nix` as the script uses `nix-instantiate`)

Reply:
> Why is bash needed?

Marking it as the interpreter with -i doesn't actually install it